source/configuring_domain/05_configuring_domain/regional_settings.rst

```diff
@@ -18,3 +18,20 @@ To change the timezone of a domain, go to Configuration > Regional settings. In
    :class: note
 
    It is also possible to directly write a city or zone in the textbox. A selection of matching existing timezones will appear.
+
+
+CSV delimiter
+-------------
+
+The preferred delimiter for CSV files, usually depending on geographical location, can either be a comma or a semicolon. The CSV delimiter configuration allows to define what delimiter will be used by default for datasets exports in CSV format.
```

Comment:
Maybe we can indicate even more explicitly that it applies to the user interface. Also, this also applies to the catalog export.

How about something like this:
> The CSV delimiter configuration allows to define what delimiter will be used when your users download a CSV export from your catalog or one of your datasets.